strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/AuthorizationConfigController.java

```diff
@@ -36,199 +48,279 @@
         extends BaseArtifactController
 {
 
+    static final String SUCCESSFUL_ADD_ROLE = "The role was created successfully.";
+    static final String FAILED_ADD_ROLE = "Role cannot be saved because the submitted form contains errors!";
+
+    static final String SUCCESSFUL_GET_CONFIG = "Everything went ok.";
```

Comment:
Is this really required?

Reply:
Required - probably not, but it's easier for making the assertion in the tests :)